sdks/python/setup.py

```diff
@@ -62,6 +62,7 @@ def get_version():
     'oauth2client>=2.0.1',
     'protorpc>=0.9.1',
     'python-gflags>=2.0',
+    'python-snappy>=0.5',
```

Comment:
As per offline discussion with @charlesccychen we reverted this change and instead made the test execute conditionally on whether snappy is installed in the system.

That way we get the best of both worlds in terms of coverage and dependencies.

@chamikaramj did you have any thoughts on the rest of the CL?